src/main/java/com/toptrumps/cli/TopTrumpsCLIApp.java

```diff
@@ -26,13 +25,13 @@ public static void main(String[] args) {
 
 
         if (writeGameLogsToFile) {
-            Logger logger = new Logger(LOG_PATH + LOG_FILENAME);
-            logger.enable();
+            Logger logger = Logger.getInstance();
```

Comment:
It could be a member variable! This way you don't need to call ```Logger.getInstance()``` all the time.

Reply:
I'll look in to it :)